evalai/utils/config.py

```diff
@@ -12,7 +12,7 @@
 
 AUTH_TOKEN_PATH = os.path.join(AUTH_TOKEN_DIR, AUTH_TOKEN_FILE_NAME)
 
-API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://localhost:8000')
+API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://evalapi.cloudcv.org')
```

Comment:
Please change it to `https://evalapi.cloudcv.org`

Reply:
@RishabhJain2018 Done!